### PR TITLE
rtmp-services: Add looch.tv streaming platform

### PR DESCRIPTION
**Pull request by pkuznets**

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -1032,6 +1032,22 @@
                     "url": "rtmp://ingest-as-central.a.switchboard.zone/live"
                 }
             ]
+        },
+        {
+            "name": "Looch",
+            "common": true,
```
Comment (derrod) [inline plugins/rtmp-services/data/services.json]: The service should not be marked as common.

**Comment by RytoEX** [comment]
You'll need to squash those commits into one commit, and the commit message should be formatted as 50/72 with the "rtmp-services" prefix.  See the [Commit Guidelines](https://github.com/jp9000/obs-studio/blob/master/CONTRIBUTING.rst#commit-guidlines) from this project's [CONTRIBUTING document](https://github.com/jp9000/obs-studio/blob/master/CONTRIBUTING.rst).

See PR #967 or PR #975 for examples of successfully merged PRs for rtmp-services.

**Comment by pkuznets** [comment]
Could you please review my pull request

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -1048,6 +1048,22 @@
                     "url": "rtmp://ingest-as-central.a.switchboard.zone/live"
                 }
             ]
+        },
+        {
+            "name": "Looch",
+            "common": false,
```
Comment (RytoEX) [inline plugins/rtmp-services/data/services.json]: I'm pretty sure you don't _have_ to specify `"common": false`.  False is assumed if the common property is not provided.

**Comment by jp9000** [comment]
Terribly sorry I haven't merged yet by the way -- I'm just finishing up something then I'll be getting back to all the pull requests.

**Pull request merged**